src/ReverseProxy/Service/RuntimeModel/Transforms/RequestHeaderOriginalHostTransform.cs

```diff
@@ -11,12 +11,16 @@ namespace Yarp.ReverseProxy.Service.RuntimeModel.Transforms
     /// </summary>
     public class RequestHeaderOriginalHostTransform : RequestTransform
     {
+        public static readonly RequestHeaderOriginalHostTransform OriginalHost = new(true);
+
+        public static readonly RequestHeaderOriginalHostTransform CustomHost = new(false);
```

Comment:
Providing a custom host is a separate feature, this one turns on or off the current host.
```suggestion
        public static readonly RequestHeaderOriginalHostTransform SuppressHost = new(false);
```